plugin/trino-redis/src/test/java/io/trino/plugin/redis/TestMinimalFunctionalityWithoutKeyPrefix.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.redis;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.redis.util.RedisServer;
+import io.trino.spi.connector.SchemaTableName;
+import io.trino.spi.type.BigintType;
+import io.trino.testing.MaterializedResult;
+import io.trino.testing.StandaloneQueryRunner;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.UUID;
+
+import static io.trino.plugin.redis.util.RedisTestUtils.createTableDescription;
+import static io.trino.plugin.redis.util.RedisTestUtils.installRedisPlugin;
+import static io.trino.plugin.redis.util.RedisTestUtils.loadSimpleTableDescription;
+import static io.trino.testing.assertions.Assert.assertEquals;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@Test(singleThreaded = true)
+public class TestMinimalFunctionalityWithoutKeyPrefix
+        extends AbstractTestMinimalFunctionality
+{
+    @Override
+    @BeforeClass
+    public void startRedis()
```

Comment:
Please remove redundant code. You can add `protected abstract Map<String, String> connectorProperties();` to `AbstractTestMinimalFunctionality` and override in `TestMinimalFunctionality` & `TestMinimalFunctionalityWithoutKeyPrefix`.